README.md

```diff
@@ -198,3 +198,17 @@ If you find a security vulnerability or any security related issues,
 please DO NOT file a public issue, instead send your report privately to
 `security@coredns.io`. Security reports are greatly appreciated and we
 will publicly thank you for it.
+
+
+### Coordination of security releases with Kubernetes project
+
+As CoreDNS is now a feature part of Kubernetes, we are organizing our security release process 
+in a way that is compatible with [Kubernetes's security release process](https://github.com/kubernetes/sig-elease/blob/master/security-release-process-documentation/security-release-process.md)
+
+1. CoreDNS Team may disclose privately the security issue in order to organize the patch and its release.
```

Comment:
The paragraph above also talks about the security process, should probably merge these two